Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -1,3 +1,4 @@
+#define UNITY_ANDROID
```

Comment:
Do we need this?

Reply:
temporary so I can write code in the class, otherwise the entire class would have no syntax highlighting

Reply:
@adrenak can we keep this for future work?

Reply:
Got it, Added a note in my TODOs to remove post testing.